staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -365,6 +374,21 @@ func (d *DiscoveryClient) SwaggerSchema(version schema.GroupVersion) (*swagger.A
 	return &schema, nil
 }
 
+// OpenAPISchema fetches the open api schema using a rest client and parses the json.
+// Warning: this is very expensive (~1.2s)
```

Comment:
Is this because of `RawMessage`, or the OpenAPI loading? Kind of worrisome depending on where we'll use it (e.g. for `get`). I remember a significant performance degradation after we switched `get` to use unstructured objects, mostly because of the json decoding to `map[string]interface{}`.